pyproject.toml

```
@@ -43,9 +43,9 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "<3.12,>=3.7"
-structlog = "^21.2.0"
+structlog = "^21.2.0,<22"
```

```suggestion
structlog = "^21.2.0"
```

I believe this is redundant.

---

Yup, you're right